### DISCUSSION TITLE
SOLWEIG (UMEP land cover grid)

### DISCUSSION THREAD

**Discussion by PasqBr** [UMEP-dev/UMEP#245] [Q&A] [answered]
Hello,

I have the following issue when i want to calculate SOLWEIG with UMEP land cover grid in QGIS:

Land cover grid includes conifer land cover class. Ground cover information (underneath canopy) is required.
Does anyone know how to fix this problem?

Cheers

**Comment by biglimp** [comment, marked as answer]
I usually reclass the conifer class to bare soil class. You can du this using the LandCoverReclassifier in QGIS

**Reply by PasqBr** [reply]
so you do it as well for decidiuous LC & grass LC class because now this appears:
"Land cover grid includes Decidiuous land cover class. Ground cover information (underneath canopy) is required."
is there not another solution? i think it is not a good solution to change vegetation LC to another one because it will influence the output.

**Reply by biglimp** [reply]
Grass is considered as ground so that I leave as it is. Under decidouos tree you can go for either grass or bare soil. 
SOLWEIG requires ground cover (what is on the ground), not land cover (what you see from above).

**Reply by PasqBr** [reply]
now i took for conifer bare soil & for deciduous grass for ground cover and it worked. thank you
